drivers/overlay/overlay_test.go

```diff
@@ -52,6 +53,47 @@ func TestContainersOverlayXattr(t *testing.T) {
 	assert.Equal(t, 0o555&os.ModePerm, fi.Mode()&os.ModePerm, root)
 }
 
+func TestSupportsShifting(t *testing.T) {
+	contiguousMap := []idtools.IDMap{
+		{
+			ContainerID: 0,
+			HostID:      1000,
+			Size:        65536,
+		},
+	}
+	nonContiguousMap := []idtools.IDMap{
+		{
+			ContainerID: 0,
+			HostID:      0,
+			Size:        1,
+		},
+		{
+			ContainerID: 2,
+			HostID:      2,
+			Size:        1,
+		},
+	}
+
+	t.Run("no mount program", func(t *testing.T) {
+		driver := graphtest.GetDriver(t, driverName)
+		defer graphtest.PutDriver(t)
```

Comment:
`GetDriver` uses `t.Cleanup()` to ensure `PutDriver` happens automatically.

(2 instances)